api/nakadi-event-bus-api.yaml

```diff
@@ -479,7 +475,40 @@ paths:
                 Number of seconds the client ought to wait before making a follow-up request.
               type: number
 
-  /event-types/{name}/partitions:
+  '/event-types/{name}/schemas':
+    get:
+      tags:
+        - schema-registry-api
+      description: |
+        List of schemas ordered from most recent to oldest.
+    responses:
+        '200':
+          schema:
+            type: array
```

Comment:
I'm uncomfortable with that because I think that every endpoint has it's peculiarities. I think that this extensibility decisions should be done only if there are envisioned extensions for the near future. Making it extensible just to be safe could lead to accidental complexity.

Are you aware of anything we might add to this endpoint in the future?